Plugins/BTCPayServer.RockstarDev.Plugins.TransactionCounter/Views/TransactionCounter/CounterConfig.cshtml

```diff
@@ -0,0 +1,169 @@
+                            </div>
+                            <div class="info-note mt-2 text-warning" role="alert">
+                                <span text-translate="true">Click the</span>
+                                <a asp-controller="PublicCounter" asp-action="Counter" target="_blank"
+                                   rel="noopener noreferrer">link</a>
+                                <span text-translate="true">to view transaction count.</span>
+                                <span text-translate="true">And on this</span>
+                                <a asp-controller="PublicCounter" asp-action="ApiCounter" target="_blank"
+                                   rel="noopener noreferrer">link</a>
+                                <span text-translate="true">to view transaction count API.</span>
+                            </div>
+
+                            <div class="subsettings">
+                                <div class="d-flex my-3">
+                                    <input asp-for="AllStores" type="checkbox" class="btcpay-toggle me-3"/>
+                                    <label asp-for="AllStores" class="form-check-label"></label>
+                                    <span asp-validation-for="AllStores" class="text-danger"></span>
+                                </div>
+                                <div class="form-group mt-3" id="store-selection"
+                                     style="display: @(Model.AllStores ? "none" : "block")">
+                                    <label asp-for="SelectedStores" class="form-check-label"></label>
+                                    @for (var i = 0; i < Model.Stores.Length; i++)
+                                    {
+                                        var store = Model.Stores[i];
+                                        var isEnabled = Model.AllStores || (Model.SelectedStores?.Any(s => s.Id == store.Id) ?? false);
+
+                                        <div
+                                            class="d-flex justify-content-between align-items-center border-bottom py-2">
+                                            <span>@store.StoreName</span>
+                                            <input type="hidden" name="SelectedStores[@i].Name"
+                                                   value="@store.StoreName"/>
+                                            <input type="hidden" name="SelectedStores[@i].Id" value="@store.Id"/>
+                                            <input type="checkbox" class="btcpay-toggle"
+                                                   name="SelectedStores[@i].Enabled" value="true"
+                                                   @(isEnabled ? "checked" : "")/>
+                                        </div>
+                                    }
+                                </div>
```

Comment:
As I said in previous comments, no need for elaborate UI logic, this will be very cumbersome on instances with 50 stores. We just provide ExcludeStoreId textarea and use that when loading data to exclude certain stores if needed.

Reply:
@TChukwuleta, to include them in the counter again, simply remove them from the `ExcludeStoreId` textarea comma-separated list, and the counter will start counting them again.